frontend/app/components/App/index.js

```diff
@@ -24,9 +24,10 @@ const App = (): React.Element<any> =>
     <Switch>
       <Route exact path="/" component={Home} />
       <Route exact path="/sessions" component={Sessions} />
+      <Route exact path="/sessions/:sessionName?/" component={Session} />
```

Comment:
The session name is not optional, otherwise this route will compete with /sessions.